packages/bits/src/lib/textbox/textbox-number/textbox-number.component.less

```diff
@@ -45,10 +46,12 @@
         height: (@nui-space-md * 2);
         top: 1px;
         text-align: center;
-        .nui & button.nui-button {
+        button.nui-button {
             padding-left: @nui-space-xs;
             width: (@nui-space-md * 2);
-            height: 28px;
+            height: @textbox-button-height;
+            min-height: @textbox-button-height;
+            padding-top: 7px;
```

Comment:
It would be good to move this to line 50 so that it's next to the other padding rule. Additionally, it would be good to use the variable `@button-padding-is-empty` which is equal to 6px and made available in buttons.less. Then, you can add 1px to that to account for the 1px height difference on the top due to overriding the default button height of 30px down to 28px.

The import for the variable would look something like:
```
@import (reference) "../../styles/components/buttons.less";
```

Reply:
```suggestion
            // adding 1px to padding to account for adjusted button height
            padding-top: @button-padding-is-empty + 1px;
```